cfgov/v1/models/base.py

```diff
@@ -110,6 +110,17 @@ class CFGOVPage(Page):
         ),
     )
 
+    is_archived = models.BooleanField(
```

Comment:
Are we supporting cases where there is a displayed archive date but also cases where we don't show the date?

To simplify the changes here, could we have a single field, maybe `archived_at` (to be consistent with Wagtail Page field `_at` [convention](https://github.com/wagtail/wagtail/blob/738003726900295328f263bfe91686026f70f596/wagtail/core/models.py#L694-L702), for example `locked_at`, `first_published_at`, etc)? This could still be checked like `if page.archived_at` if it defaults to null.

Reply:
@chosak That is a great question. We've recommended displaying the date a page was archived, but I don't think we've gotten feedback on that specifically so I'm not sure whether that could be subject to change. I can try to find out. In any case, changing this field to `archived_at` to follow Wagtail convention makes sense.

We may also need to add more fields here as there will likely be a need for different archiving text on different pages.

Reply:
Updated field to `archived_at` in https://github.com/cfpb/consumerfinance.gov/pull/6015/commits/18c4814820c56409838615d646942e2c8afecb56 -- thanks for the suggestion, @chosak!

Reply:
Following up -- my team discussed and we're not sure whether we can rely on the presence of an archived_at date, so we think it would make sense at this point to keep the is_archived field.